Library/Homebrew/config.rb

```diff
@@ -50,7 +50,11 @@ def mkpath
 HOMEBREW_TEMP = Pathname.new(ENV.fetch("HOMEBREW_TEMP", "/tmp"))
 
 unless defined? HOMEBREW_LIBRARY_PATH
-  HOMEBREW_LIBRARY_PATH = Pathname.new(__FILE__).realpath.parent.join("Homebrew")
+  HOMEBREW_LIBRARY_PATH = Pathname.new(__FILE__).realpath.parent
+end
+
+unless defined? HOMEBREW_CABAL_CONFIG_PATH
```

Comment:
Is there a reason it couldn't live directly in `language/haskell.rb`?

My point about `config` was just that the variables here primarily directly impact how `brew` operates, not `brew` + a function, in this situation, `brew install xyz` where `xyz` is a Haskell-using formulae.